src/meta/service/src/ddl_service.rs

```diff
@@ -170,17 +154,14 @@ impl DdlService for DdlServiceImpl {
         request: Request<CreateSchemaRequest>,
     ) -> Result<Response<CreateSchemaResponse>, Status> {
         let req = request.into_inner();
-        let id = self.gen_unique_id::<{ IdCategory::Schema }>().await?;
```

Comment:
Moved `gen_unique_id` to ddl controller, they will be called only in metadata manager V1 version.